src/mpcCoreKit.ts

```diff
@@ -116,6 +116,7 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     if (typeof options.manualSync !== "boolean") options.manualSync = false;
     if (!options.web3AuthNetwork) options.web3AuthNetwork = WEB3AUTH_NETWORK.MAINNET;
     if (!options.sessionTime) options.sessionTime = 86400;
+    if (!options.serverTimeOffset) options.serverTimeOffset = 0; // ms
```

Comment:
we need to pass this param in torus.js and customAuth constructor

Reply:
Made a PR, https://github.com/torusresearch/CustomAuth/pull/331 to include `serverTimeOffset` value in respective constructors.

Reply:
okay, till we merge and release custom auth, can you remove serverTimeOffset param from this sdk